bootstring/src/main/scala/cats/bootstring/Base.scala

```diff
@@ -56,62 +98,79 @@ object Base {
         codePoint - 22
       } else {
         throw new IllegalArgumentException(
-          s"Code point $codePoint is valid for the given base.")
+          s"Code point $codePoint is not valid for the given base.")
       }
 
-    unsafeFrom(36, lowercaseChars, codePointToDigit)
-  }
+    // Developer's note,
+    //
+    // This method exists because on ScalaJS we can not catch the array out of
+    // bounds exception.
+    //
+    // See https://www.scala-js.org/doc/semantics.html#undefined-behaviors
+    def intToCodePointDigit(int: Int, arr: Array[Int]): Int =
+      if (int < arr.size) {
+        arr(int)
+      } else {
+        throw new RuntimeException(s"There is no digit in this base which corresponds to $int.")
```

Comment:
Why not an illegal argument exception?

Reply:
No particular reason. IllegalArgumentException is better, I'll update it.